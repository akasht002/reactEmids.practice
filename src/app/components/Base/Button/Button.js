import React from 'react';

class Button extends React.Component {
    render() {
        return(
            <button
                type={this.props.type}
                className={this.props.classname}
                onClick={this.props.onClick}
                disabled={this.props.disable}>
                {this.props.label}
            </button>
        );
    }
}

export default Button;