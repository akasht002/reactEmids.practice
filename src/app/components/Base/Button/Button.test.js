import React from 'react';
import Button from './Button';
import renderer from 'react-test-renderer';

test('Button Component Jest', () => {
  const component = renderer.create(
    <Button type='button' classname='btn' disable='true'></Button>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

