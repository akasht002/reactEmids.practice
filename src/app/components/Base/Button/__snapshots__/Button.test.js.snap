// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button Component Jest 1`] = `
<button
  className="btn"
  disabled="true"
  type="button"
/>
`;
